src/context.ts

```diff
@@ -24,6 +24,18 @@ export default class Context {
   contexts: Contexts;
 
   constructor(contexts: Contexts) {
+    if (!Object.values(contexts).every((item: any) => typeof item === "object")) {
+      console.error("Context must be an array of objects");
```

Comment:
I think this message might be misleading.

The format we expect is something like

```javascript
{
  user: {
    email: "test@example.com",
  },
  device: { mobile: true },
}
```

Right?

That's not an array of objects, but an object with two keys, `user` and `device`, each with an object as value.

I think your use of `Object.values` led to the misleading messaging.

Reply:
Ah, yes, thank you. Iterating over `Object.values` (an array) is exactly why I made that error. Will fix the message.